docs/versioned_docs/version-2.8/20-usage/40-secrets.md

```diff
@@ -32,7 +38,28 @@ The case of the environment variables is not changed, but secret matching is don
 
 You can set an setting or environment value from secrets using the `from_secret` syntax.
 
-In this example, the secret named `secret_token` would be passed to the setting named `token`,which will be available in the plugin as environment variable named `PLUGIN_TOKEN` (See [plugins](./51-plugins/20-creating-plugins.md#settings) for details), and to the environment variable `TOKEN_ENV`.
+The example below passes a secret called `token` as an environment variable that will be called `ENV_TOKEN`:
+
+```diff
+ steps:
+   - name: docker
+     image: my-plugin
```

Comment:
[Not a plugin](https://github.com/woodpecker-ci/woodpecker/pull/4561/files#r1883397580)

Reply:
Thanks for the suggested edits and spotting those mistakes! 

I'm not sure how to address this/to what to change the plugin bits in here though, as I just kept what was already in the docs. Do you have a suggestion? :)

Reply:
```diff
 steps:
   env-token-example:
     image: alpine
     commands:
+      - echo "The secret is $TOKEN_ENV"
+    environment:
+      TOKEN_ENV:
+        from_secret: secret_token
```